ipmiMgrApp/Db/Makefile

```diff
@@ -16,6 +16,7 @@ DB += sensor_ai_alias.db
 DB += fru_common_alias.db
 DB += fru_cu_common_alias.db
 DB += system_common_alias.db
+DB += power_mngr.db
```

Comment:
Should this be `power_module.db`? I could not find a `power_mngr` template in [slac-epics-modules/ipmiComm](https://github.com/slac-epics-modules/ipmiComm) nor a substitution with this name.